tests/js/client/aql/aql-query-cache-noncluster.js

```diff
@@ -86,12 +64,7 @@ function ahuacatlQueryCacheTestSuite () {
 
       c1 = null;
       c2 = null;
-
-      let command = `
-        AQL_QUERY_CACHE_PROPERTIES(${JSON.stringify(cacheProperties)});
-        AQL_QUERY_CACHE_INVALIDATE();
-      `;
-      arango.POST("/_admin/execute", command);
+      cache.properties(cacheProperties);
```

Comment:
Should we also add a `cache.clear();` here, so the behavior is equivalent with the previous version?